packages/sign_in_with_apple/lib/sign_in_with_apple.dart

```diff
@@ -21,35 +32,137 @@ export './src/widgets/sign_in_with_apple_button.dart'
 // ignore: avoid_classes_with_only_static_members
 class SignInWithApple {
   @visibleForTesting
-  static const channel =
-      MethodChannel('de.aboutyou.mobile.app.sign_in_with_apple');
+  static const channel = MethodChannel(
+    'com.aboutyou.dart_packages.sign_in_with_apple',
+  );
+
+  /// Returns the credentials stored in the Keychain for the app-associated websites
+  static Future<AuthorizationCredentialPassword> getKeychainCredential() async {
```

Comment:
I guess with this change we would consider this a breaking change.

Let's release v2 maybe after we fixed the feedback?